samples/nextjs/src/pages/[[...path]].tsx

```diff
@@ -58,12 +57,12 @@ export const getStaticPaths: GetStaticPaths = async (context) => {
   // See https://nextjs.org/docs/basic-features/data-fetching#incremental-static-regeneration
 
   if (process.env.NODE_ENV !== 'development') {
-    const ROOT_ITEM = `/sitecore/content/${config.jssAppName}/home`;
-    const paths = await graphQLSitemapService.fetchSSGSitemap(context.locales || [], ROOT_ITEM);
+    // Note: Next.js runs export in production mode
+    const paths = await sitemapFactory.create(context);
 
     return {
       paths,
-      fallback: 'blocking',
+      fallback: false,
```

Comment:
Looks like this was unintentionally switched back to `false` (recent fix set it to 'blocking').

Reply:
Oh, I see you're depending on it being `false` for export (based on doc change). However, this needs to be 'blocking' for non-export due to open issue (see #551). We need to check PHASE_EXPORT here.

Reply:
@ambrauer Look: https://github.com/Sitecore/jss/pull/555#discussion_r572665191
Added check for `EXPORT_MODE`